### PR TITLE
Add NOTICE.txt, and replace LICENCE.md with LICENSE.txt

### PR DESCRIPTION
**Pull request by flimzy**
Relates to #418

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -16,3 +16,25 @@ to this project are therefore also contributed under the same license. **If you
 are uncomfortable with, or unable to comply with this** (perhaps due to legal
 limitations imposed by your employer), **DO NOT SUBMIT CODE**! You are still
 welcome to participate in bug reports and discussions.
+
+### Copyright headers
+
+New files you contribute should use the standard copyright header:
+
+```
+Copyright 2019 Kivik Contributors
```
Comment (kirillDanshin) [inline CONTRIBUTING.md]: 2017-2019

Comment (flimzy) [reply CONTRIBUTING.md]: Is it really valid to pre-date a copyright before a thing existed?

I can't find a definitive answer for the Apache license, so I followed [Go's lead](https://golang.org/doc/contribute.html#copyright).

Comment (kirillDanshin) [reply CONTRIBUTING.md]: @flimzy both ways are actually valid, but speaking of validation of file headers, it's much easier to validate license without having to check when each file was created

Comment (flimzy) [reply CONTRIBUTING.md]: A fair point. On the other hand, it's annoying to do a mass update of every file every year in January. I'll consider which approach to take. Thanks for taking the time to consider my project :)

**Comment by flimzy** [comment]
Resolved in #503

**Pull request closed without merging**